### ISSUE TITLE
Function call arguments on multiple lines

### ISSUE THREAD

**Issue by affanshahid** [prettydiff/prettydiff#315]
I can't seem to find an option which allows me to have different arguments on difference lines. Sort of like the the "leave method chains alone" for the methodchain option. So that the following input would cause no changes.

input: 
```
ReactDOM.render(<Voting hasVoted='Trainspotting' winner='Trainspotting' pair={pair}/>,
document.getElementById('app'))
```

actual output:
```
ReactDOM.render(
    <Voting hasVoted='Trainspotting' winner='Trainspotting' pair={pair}/>, document.getElementById('app'));
```

**Comment by prettydiff** [comment]
I will have to look into this.  At the very least it would be nice if long listed arguments wrap.  This is probably a missing feature opposed to a defect, but I will investigate.

**Comment by prettydiff** [comment]
This is fixed in branch 2.0.4.  So, far this requires that the total size of all method arguments exceed the value of option `wrap` and I am only applying it to JSX code.
